docker-compose.yml

```diff
@@ -27,11 +27,16 @@ services:
       # sparkly new oauth2
       - TWITTER_OAUTH2_CLIENT_ID=
       - TWITTER_OAUTH2_CLIENT_SECRET=
+      - HIKARI_MAXIMUM_POOL_SIZE=1000
```

Comment:
1000 is a lot of concurrent conenctions, do you really need that many?
https://github.com/brettwooldridge/HikariCP/wiki/About-Pool-Sizing has some interesting reading material